docs_src/getting-started/Ch-GettingStartedSnapUsers.md

```diff
@@ -0,0 +1,127 @@
+# Getting Started - Snap Users
```

Comment:
Maybe call this one just "Snap" and the other one just "Docker"

Reply:
Getting Started with Snaps and Getting Started with Docker Containers??